pkg/ruler/rules/local/local.go

```diff
@@ -0,0 +1,136 @@
+package local
+
+import (
+	"context"
+	"flag"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
+	rulefmt "github.com/cortexproject/cortex/pkg/ruler/legacy_rulefmt"
+	"github.com/cortexproject/cortex/pkg/ruler/rules"
+)
+
+type Config struct {
+	Directory string `yaml:"directory"`
+}
+
+// RegisterFlags registers flags.
+func (cfg *Config) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
+	f.StringVar(&cfg.Directory, prefix+"local.directory", "", "Directory to scan for rules")
+}
+
+// Client expects to load already existing rules located at:
+//  cfg.Directory / userID / namespace
+type Client struct {
+	cfg Config
+}
```

Comment:
nit: This can be named RuleStore to be consistent with the obj store naming convention